### ISSUE TITLE
Browning

### ISSUE THREAD

**Issue by mnk77** [obophenotype/mammalian-phenotype-ontology#2632]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
Browning

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Conversion of white adipocytes to a morphology more closely resembling brown adipocytes (eg. multilocular droplets, decreased lipid droplet size).

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
PMID 24531262

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
abnormal white adipose tissue morphology MP:0002970 

I'm not sure how this fits in with beige fat cells.

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
MGI:mperry

**Comment by anna-anagnostop** [comment]
A definition for white adipose tissue browning is provided in PMID:27397607 (glossary), see below:

white adipose tissue browning
DEF: the emergence of brown adipocytes and or the conversion of white adipocytes into beige adipocytes in white adipose tissue (WAT) – a process that represents adaptation to increased thermogenic demand, exercise, injury (thermal injury), and disease (cancer)
DEF SOURCE: PMID:27397607 (glossary)
SYN: browning of white adipose tissue
SYN: browning of WAT
SYN: WAT browning
SYN: beiging of WAT
SYN: beiging of white adipose tissue
SYN: WAT beiging


We already have terms for abnormal beige fat cell morphology MP:0020325 and abnormal beige fat cell physiology MP:0020426

We should probably amend their definition to reflect that inducible beige fat cells can develop in WAT depots in response to specific stimuli, such as chronic cold exposure or beta-adrenergic stimulation; similar to brown adipocytes in BAT, beige cells in white adipose tissues have the ability to burn fat and are characterized by a multilocular/small lipid droplet structure, high amounts of mitochondria and UCP1 expression
SOURCE: PMID:26362189

Anna

**Comment by mnk77** [comment]
Amending the definitions for the beige cells would be beneficial. I wasn’t certain whether white adipose tissue that underwent browning would be considered beige cells or if they exist independent of browning.

Michelle

**Comment by sbello** [comment]
MP:0020521
For both this and the browning term I used browned and whitened to clarify that this is the morphological consequence of the process not the process itself. I modified the definitions in a similar manner. I felt whitening and browning came across as the process rather than the endpoint. I did include these are synonyms.
